regression-tests/pure2-print.cpp2

```diff
@@ -5,7 +5,7 @@ testing_enabled: bool = false;
 
 outer: @print type = {
 
-    object_alias: <T> T requires true == 42;
+    object_alias: <T> T requires true == sizeof(type T::value_type);
```

Comment:
Is this just to exercise `type` for this pretty-print self-test, or is `type` now intended to be required here?

Reply:
The former.
This change is intended to exercise the `type` keyword in the pretty-print.